docs/discover/field-statistics.asciidoc

```diff
@@ -0,0 +1,62 @@
+[[show-field-statistics]]
+== Explore the fields in your data
+
+beta::[]
+
+Explore the fields in your data with the *Field statistics* view in *Discover*.
+You can answer questions such as:
+
+* What does the latency
+look like when one of the containers is down on a Sunday?
+* How many fields in my index are empty or redundant?
```

Comment:
@qn895 not sure Field statistic can help to find out which fields are empty or redundant?

Reply:
@kertal Can you suggest another example to add here?

Reply:
@kertal This is a good point, as it is in Discover, the empty fields are disabled/not visible in Discover 🤔 We can definitely remove this line here @gchaps